de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/ddl/functions/exportSalesInvoicesInDateRange.sql

```diff
@@ -0,0 +1,38 @@
+DROP FUNCTION IF EXISTS exportSalesInvoicesInDateRange(TIMESTAMP With Time Zone, TIMESTAMP With Time Zone, numeric);
+
+CREATE OR REPLACE FUNCTION exportSalesInvoicesInDateRange(IN p_C_Invoice_dateinvoiced_start TIMESTAMP With Time Zone, 
+														  IN p_C_Invoice_dateinvoiced_end TIMESTAMP With Time Zone,
+														  IN p_AD_Org_ID numeric)
+
+    RETURNS TABLE
+            (
+                AD_Org_ID character varying,
+				C_Bpartner_ID character varying,
+				C_Invoices_Count bigint
+            )
+AS
+
+$BODY$
+
+SELECT DISTINCT org.name as AD_Org_ID,
+	bp.name as C_Bpartner_ID,
+	(SELECT COUNT(*)
+	 FROM   C_Invoice other
+	 WHERE other.c_bpartner_id = inv.c_bpartner_id
+ 	 AND other.ad_org_id = inv.ad_org_id
+	 AND other.dateinvoiced >= p_C_Invoice_dateinvoiced_start
+     AND other.dateinvoiced <= p_C_Invoice_dateinvoiced_end
+	)
+	 AS C_Invoices_Count
+	
+FROM C_Invoice inv
+	LEFT OUTER JOIN ad_org org on org.ad_org_id = inv.ad_org_id
```

Comment:
This is a misunderstanding: the org-partner means the BPartner that "represents" the org...not the invoice's partner.

So basically, if you write a sales-invoice with Org=ABC, then
* the BPartner from `inv.c_bpartner_id` is supposed to pay to Org ABC
* and ABC's "organization-bpartner" will find the paid money on their bank-accound

So, to get from an invoice to the "organization-bpartner", you go `C_Invoice.AD_Org_ID` and then `AD_Org.AD_OrgBP_ID`.